src/components/ScreeningCard.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+    <div class="screening-card">
+        <div class="screening-card__image" >
+            <img :src="movie.poster_url" alt="movie-image" />
+        </div>
+        <div class="screening-card__info">
+            <h3 class="screening-card__title">{{movie.title}}</h3>
+            <div class="screening-card__details">
+                <div class="screening-card__genre">{{movie.genre.name}}</div>
+                <div class="screening-card__length">{{setLength(movie.length)}}</div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+
+export default {
+    name: 'ScreeningCard',
+    props:{
+      movie: {
+      type: Object,
+      required: true,
+    },
+    },
+
+    methods:{
+    setLength(inMinutes){
+      const hours = Math.floor(inMinutes/60);
+      const minutes = `${inMinutes%60}`.slice(-2);
+
+      return `${hours}h ${minutes} min`
+
+  }
+    }
+}
+</script>
+
+<style lang="scss">
+
+@media(min-width: 0px){
```

Comment:
When I remove this query (min-width: 0px) styles of image don't behave properly - so I think that maybe in this case I will leave it as it is...

Reply:
Ok, but IMO it cannot be related to presence or not of this media query..
Maybe it's because of anothwer problem e.g. you have misplaced brackets in your styles:
```
@media(min-width: 0px){
  // ...
  @media(min-width: 690px){
    // ...
  }
}
```

and it should be
```
@media(min-width: 690px){
  // ...
}
@media(min-width: 690px){
  // ...
}
```

is it possible that you removed the wrong ending bracket by accident?